src/setupTests.js

```diff
@@ -1,4 +1,6 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
```suggestion
// eslint-disable @typescript-eslint/no-var-requires
```
I think you could reasonably make it a single file-wide configuration.